pkg/interface/src/views/apps/dojo/app.js

```diff
@@ -20,7 +20,7 @@ export default class DojoApp extends Component {
     this.store.setStateHandler(this.setState.bind(this));
 
     this.state = this.store.state;
-    this.resetControllers();
+    this.resetControllers = this.resetControllers.bind(this);
```

Comment:
This bind is unnecessary (a no-op, given that the function is not used in any closures that would otherwise lose the `this` context)

Reply:
so it is ...